includes/constants.php

```diff
@@ -68,6 +68,8 @@
 define( 'ACTIVITYPUB_BLOG_MODE', 'blog' );
 define( 'ACTIVITYPUB_ACTOR_AND_BLOG_MODE', 'actor_blog' );
 
+define( 'ACTIVITYPUB_ACTOR_POST_TYPE', 'ap_actor' );
```

Comment:
So far we've used class constants for cpt slugs, is there a way to continue with that? Maybe in `Activitypub\Activity\Actor`?

Reply:
To be consistent we would have to use the collection, but yes, valid point... also thought about that. Would love to get rid of the models too. Should be possible to use the actor object.